### ISSUE TITLE
Does not play subtitles embedded in MKV

### ISSUE THREAD

**Issue by talesam** [jellyfin/jellyfin-tizen#227]
I installed it on my Samsung Freestyle projector, everything works fine, the subtitles that are embedded in the MKV files appear fine, but they are not activated when selected. This only happens with this app, in the Android version for cell phones and Android TV it works perfectly.

**Comment by dmitrylyzo** [comment]
Could you elaborate "not activated"?
What is the type of subtitles?

**Comment by talesam** [comment]
I recorded the screen of my projector, which has the Tizen app, and I recorded it on my cell phone, the same film, the subtitles appear, on Tizen the subtitles are there, but they don't appear.

https://cloud.talesam.org/s/gMmfLwRdHBEqjqq
https://cloud.talesam.org/s/35K5fBpKHtTNqiy

**Comment by dmitrylyzo** [comment]
So this is SRT.

Extracting subtitles may take a while. They should appear in `<jellyfin_data_folder>/subtitles/...`

Is there any error in the logs?
Do they work if you extract them manually and place them next to the media item (in the same folder)?

**Comment by talesam** [comment]
> So this is SRT.
> 
> Extracting subtitles may take a while. They should appear in `<jellyfin_data_folder>/subtitles/...`
> 
> Is there any error in the logs? Do they work if you extract them manually and place them next to the media item (in the same folder)?

Yes, if I put the srt file together with the video file it loads instantly. But like I said, on Android it works normally, like in the video I made, and in the Tizen version the subtitles appear to be selected, but they don't load.

**Comment by dmitrylyzo** [comment]
> But like I said, on Android it works normally

External player on Android specifies that it supports embedded subtitles. In this case it can play instantly. Not sure about Exo.
Tizen currently requires subtitles to be external (extracted).

> in the Tizen version the subtitles appear to be selected, but they don't load.

AFAIK, the bigger file, the longer it takes to extract.
Have you look if they extracted to where I said? And errors in logs?

There is `Subtitle Extract` plugin that should extract all subtitles in advance, but I haven't used it.
